dbt_project.yml

```diff
@@ -26,8 +26,8 @@ vars:
     jira_using_sprints: true # disable if you are not using sprints in Jira
     jira_using_components: true # disable if you are not using components in Jira
 
-  dbt_utils_dispatch_list: [spark_utils, fivetran_utils]
-  fivetran_utils_dispatch_list: [spark_utils]
+  # dbt_utils_dispatch_list: [spark_utils, fivetran_utils]
+  # fivetran_utils_dispatch_list: [spark_utils]
```

Comment:
```suggestion
```
This functionality has actually been fully depreciated by dbt-labs in dbt 0.20.0 so we can remove this instead of commenting out.